docs/content/users/quickstart.md

```diff
@@ -326,16 +326,23 @@ ddev launch
 You can set up a Shopware 6 environment many ways, we recommend the following technique:
 
 ```bash
-git clone --branch=6.4 https://github.com/shopware/production my-shopware6
-cd my-shopware6
-ddev config --project-type=shopware6 --docroot=public
-ddev start
-ddev composer install --no-scripts
+mkdir my-shopware6 && cd my-shopware6
+ddev config --project-type=shopware6 --docroot=public --create-docroot
+ddev composer create shopware/production:^v6.5
+# If it asks `Do you want to include Docker configuration from recipes?` answer `x`, as we're setting it up.
 # During system:setup you may have to enter the Database user (db), Database password (db)
 # Database host (db) and Database name (db).
-ddev exec bin/console system:setup --database-url=mysql://db:db@db:3306/db --app-url='{DDEV_PRIMARY_URL}'
+
+# When this asks for database configuration, use the following:
+# Database user: [db]
+# Database password: [db]
+# Database host: [db]
+# Database name: [db]
+ddev exec bin/console system:setup --database-url=mysql://db:db@db:3306/db --app-url='${DDEV_PRIMARY_URL}'
```

Comment:
I needed to move `.env` to `.env.local` to be able to run this command:

```
$ ddev exec bin/console system:setup --database-url=mysql://db:db@db:3306/db --app-url='${DDEV_PRIMARY_URL}'
Warning: Failed to load plugins. Message: An exception occurred in the driver: SQLSTATE[HY000] [2002] No such file or directory

                                                                                
 [WARNING] It looks like you have installed Shopware with Symfony Flex. You     
           should use a .env.local file instead of creating a complete new one  
                                                                                

Shopware setup process
======================

 This tool will setup your instance.

 // Instance has already been set-up. To start over, please delete your .env    
 // file.
```

All good:
```
$ mv .env .env.local
$ ddev exec bin/console system:setup --database-url=mysql://db:db@db:3306/db --app-url='${DDEV_PRIMARY_URL}'
...
Check if everything is ok. Write into "/var/www/html/.env"? (yes/no) [no]:
 > yes

 ! [NOTE] Writing into /var/www/html/.env
```

Reply:
`ddev start` should have created the .env.local for you, and the system:setup doesn't allow .env to pre-exist in this version. So if the .env.local wasn't created, there's something wrong.

Reply:
I don't see `ddev start` instruction. I can try again with `ddev start` after `ddev config`.

Reply:
Tried it again with `ddev start`, but `.env.local` was not created on `ddev start`.